README.md

```diff
@@ -1 +1,50 @@
 # authboot
```

Comment:
Should this have some discussion of what this component is, why I'd use it, etc?

Reply:
I think so. The below doesn't talk about all just a `lookup` function which could mean anything.  Maybe cover some of your response [here](https://github.com/warehouseai/authboot/pull/1/files#r174269799) as well.